docs/source/methodology.rst

```diff
@@ -0,0 +1,286 @@
+
+An octree mesh is a discretization of a 3D volume into a set of rectangular cells. The cells are defined by a tree
+structure, where each node has 8 children.
+The tree is grown by splitting each node into 8 children, until a minimum cell size is reached.
+The figure below shows an example of an octree mesh along a cross-section.
+
+.. figure:: /images/octree_grid.png
+    :width: 800
+
+We define cells with the same size as a ``level`` of the tree, where the smallest cell size corresponds to Level 1.
+
+The entire octree creation process can be broken down into two main steps:
+
+- `Creating the base mesh (General Parameters) <mesh_creation>`_ that defines the outer extents, core cell and padding parameters.
+- `Refinining the grid (Optional Parameters) <refinement>`_ to increase the resolution of the mesh in specific regions based on a set of rules.
+
+.. _mesh_creation:
+
+Creating the base
+-----------------
+
+This step relies on the
+`discretize.utils.mesh_builder_xyz <http://discretize.simpeg.xyz/en/main/api/generated/discretize.utils.mesh_builder_xyz.html?highlight=xyz#discretize-utils-mesh-builder-xyz>`_
+method to create the base mesh (single large box). The following general parameters provide controls on the position and
+outer limits of the mesh.
+
+
+.. figure:: /images/extent_parameters.png
+    :width: 800
+
+- **Core hull extent**:
+
+    List of objects available in the target ``geoh5`` project. The base mesh will be centered on the
+    selected object and fully containing its hull. If the mesh is to be used towards a forward simulation or an inversion,
+    the object would be the survey entity.
+
+- **Minimum depth**:
+
+    Thickness of the mesh added below the core region. This is useful to ensure
+    a minimum thickness of the mesh that extends below the lowest point of the input object.
+
+- **Core cell size**:
+
+    - *Easting (m)*: Smallest cell size along X-axis, in meters.
+    - *Northing (m)*: Smallest cell size along Y-axis, in meters.
+    - *Vertical (m)*: Smallest cell size along Z-axis, in meters.
+
+- **Padding distance**:
+
+    - *Horizontal (m)*: Distance to extend the mesh along the XY-plane.
+    - *Vertical (m)*: Distance to extend the mesh above and below the core + minimum depth.
+
+    .. note::
+        Setting the *vertical padding = 0* will place the top of the mesh at the highest elevation of **Core hull extent** object.
+        This is useful for creating a mesh that minimizes the amount of air cells.
+
+.. _mimimum_refinement:
+
+- **Minimum Refinement**:
+
+    Largest octree level allowed after refinement.
+    The equivalent cell dimension is given by:
+
+    .. math::
+
+        h \times 2^{level - 1}
+
+    where *h* is the *core cell size* in a given direction.
+
+Example
+^^^^^^^
+
+The example below demonstrates this process with simple line survey shown below and the following parameters:
+
+.. image:: images/octree_padding_distance.png
+    :width: 800
+    :alt: paddings
+
```

Comment:
This figure should have a caption making clear that the top image is plan view and bottom is section viewed from south (axes are hard to see)

Reply:
Roger